lucene/CHANGES.txt

```diff
@@ -40,6 +40,12 @@ Improvements
   docs on equal scores. Also, remove the ability of TopDocs.merge to set shard
   indices (Atri Sharma, Adrien Grand, Simon Willnauer)
 
+* LUCENE-8874: Show SPI names instead of class names in Luke Analysis tab. (Tomoko Uchida)
+
+* LUCENE-8894: Add APIs to find SPI names for Tokenizer/CharFilter/TokenFilter factory classes. (Tomoko Uchida)
```

Comment:
Same here ?

Reply:
@jimczi Thanks for pointing it out, was a conflict merge error. Fixed now